Grand Circus - OOP/GrandCircus/CircusModel/Circus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using iQuest.GrandCircus.Presentation;
+using System.Collections.Generic;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal class Circus
+    {
+        List<IAnimal> animals=new List<IAnimal>();
+        public Circus(Arena arena)
+        {
+            animals.Add(new Lion("Mufasa"));
```

Comment:
Instead of initializing the list field upon declaration, and then add each individual animal, we can initialize it from the constructor with all the animals:
 animals = new List<IAnimal>
            {
                new Lion("Mufasa"),
                new Bear("Baloo"),
                new Snake("Ka"),
                new Elephant("Admiral")
            };
        }